# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../audacious-plugins/gcc43.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- audacious-plugins-1.3.5/src/modplug/archive/arch_gzip.cxx.orig	2010-11-19 15:55:07.000000000 -0330
+++ audacious-plugins-1.3.5/src/modplug/archive/arch_gzip.cxx	2010-11-19 15:56:40.000000000 -0330
@@ -10,6 +10,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <stdio.h>
+#include <string.h>
 
 #include "arch_gzip.h"

--- audacious-plugins-1.3.5/src/modplug/archive/arch_rar.cxx.orig	2010-11-19 16:04:38.000000000 -0330
+++ audacious-plugins-1.3.5/src/modplug/archive/arch_rar.cxx	2010-11-19 16:06:18.000000000 -0330
@@ -10,6 +10,8 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
 
 #include "arch_rar.h"
 #include <iostream>

--- audacious-plugins-1.3.5/src/modplug/archive/arch_raw.cxx.orig	2010-11-19 15:40:52.000000000 -0330
+++ audacious-plugins-1.3.5/src/modplug/archive/arch_raw.cxx	2010-11-19 15:42:40.000000000 -0330
@@ -11,6 +11,7 @@
 //mmap()
 #include <unistd.h>
 #include <sys/mman.h>
+#include <stdlib.h>
 
 #include "arch_raw.h"

